.github/workflows/deploy-cloud-nodes.yml

```diff
@@ -64,7 +64,14 @@ jobs:
             docker pull registry.digitalocean.com/magmo/go-nitro:latest
             docker stop $NODE_NAME || true
             docker rm $NODE_NAME || true
-            docker run --restart=unless-stopped -it -d --name $NODE_NAME -p 3005:3005 -p 4005:4005 -p 80:5005 -e NITRO_CONFIG_PATH=$NITRO_CONFIG_PATH -e SC_PK=$SC_PK -e CHAIN_PK=$CHAIN_PK -v /var/nitro_store:/app/data registry.digitalocean.com/magmo/go-nitro:latest
+            docker run --restart=unless-stopped -it -d --name $NODE_NAME \
+              -p 3005:3005 -p 4005:4005 -p 80:5005 \
+              -e NITRO_PUBLIC_IP="0.0.0.0"
```

Comment:
Curious, why is the public address set to "0.0.0.0"?

Reply:
Looks like this might have been addressed in https://github.com/statechannels/go-nitro/pull/1711/commits/5fc35f02bfc3c36aa6684a30c0f96624083921d2?

Reply:
That was an artifact of some troubleshooting I was doing. That line has been changed to the following:
```
              -e NITRO_PUBLIC_IP=$DROPLET_IP
```